src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/McPkg.cs

```diff
@@ -6,6 +6,7 @@
 using Equinor.ProCoSys.Common.Time;
 using Equinor.ProCoSys.Common;
 using Equinor.ProCoSys.IPO.Domain.AggregateModels.CertificateAggregate;
+using System.ComponentModel.DataAnnotations.Schema;
```

Comment:
Undo all changes in this file, line 9, 65 and 66
The file should not appear as changed here in PR